### PR TITLE
Han dynasty guards

### PR DESCRIPTION
**Pull request by indoptogopt**
Do you think the mentioning the Chinese names fits? 
Would it be better without the exact year numbers?

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 1]
```diff
@@ -6,6 +6,12 @@
     <SpecificName>Hǔ Bēn</SpecificName>
     <SelectionGroupName>units/han/champion_infantry_spearman</SelectionGroupName>
     <Icon>units/han/champion_infantry_swordsman.png</Icon>
+    <History>
+        The Han empire's capital, Chang'an (206 BC to 9 AD, 190 BC to 195 BC), later Louyang (23–190 AD, 196 AD), and Xuchang (196–220 AD), was protected by various guard corps.
+        The largest and strategically most important was the Northern Army, which was stationed in the surroundings of the capital. It consisted of five regiments: the Archers, the Footsoldiers, the Elite Cavalry, the Garrison Cavalry, and the Chang River Regiment. Latter comprised non-Chinese auxiliary troops. Each regiment contained around 1,000 professionally trained soldiers. There was also a Southern Army of similar size; however, it lasted shorter and was of lower importance.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:11]:
```suggestion
        The largest and strategically most important was the Northern Army.  It consisted of five regiments: the Archers, the Foot soldiers, the Elite Cavalry, the Garrison Cavalry, and the Chang River Regiment. Later, the army also contained non-Chinese auxiliary troops. Each regiment contained around 1,000 professionally trained soldiers. There was also a Southern Army of similar size; however, it didn't last as long and was of lower importance.
```

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 2]
```diff
@@ -6,6 +6,12 @@
...
+        The palace guards patrolled the capital's multiple palaces' gates, towers, and walls. They numbered around 2000 and were managed by the Minister of Guards, one of the nine ministers.The Minister of the Household commanded several guard squads as well. The two militarily most valuable were the "虎賁 huben," Rapid as Tigers (which the Champion Spearman represents), and the "羽林 yulin," Feathered Forest. Both were between 1000 and 2000 men strong and responsible for the emperor's immediate personal security. Memberships in these units were passed on to the next generation.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:12]: The Chinese names are unfortunately kind of distracting.

**Review by TheShadowOfHassen — COMMENTED** [review]
Content edit.

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml (hunk 2 above)
Comment (indoptogopt) [reply 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:12]: Do you only mean the Chinese words or also the Latinized transcriptions?

Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:12]: Everything in the quotations

Comment (indoptogopt) [reply 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:12]: okay

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 3]
```diff
@@ -6,6 +6,12 @@
     <SpecificName>Hǔ Bēn</SpecificName>
     <SelectionGroupName>units/han/champion_infantry_spearman</SelectionGroupName>
     <Icon>units/han/champion_infantry_swordsman.png</Icon>
+    <History>
+        The Han empire's capital was protected by various guard corps.
+        The largest and strategically most important was the Northern Army. It consisted of five regiments: the Archers, the Foot soldiers, the Elite Cavalry, the Garrison Cavalry, and the Chang River Regiment. Later, the army also contained non-Chinese auxiliary troops. Each regiment contained around 1,000 professionally trained soldiers. There was also a Southern Army of similar size; however, it didn't last as long and was of lower importance.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:11]:
```suggestion
        The largest and strategically most important was the Northern Army. It consisted of five regiments: the Archers, the Foot soldiers, the Elite Cavalry, the Garrison Cavalry, and the Chang River Regiment. Later, the army also contained non-Chinese auxiliary troops. Each regiment contained around 1,000 professionally trained soldiers. There was also a Southern Army of similar size, however, it didn't last as long and was of lower importance.
```

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 4]
```diff
@@ -6,6 +6,12 @@
...
+        The palace guards patrolled the capital's multiple palaces' gates, towers, and walls. They numbered around 2000 and were managed by the Minister of Guards, one of the nine ministers.The Minister of the Household commanded several guard squads as well. The two militarily most valuable were the “Rapid as Tigers” (which the Champion Spearman represents), and the “Feathered Forest”. Both were between 1000 and 2000 men strong and responsible for the emperor's immediate personal security. Memberships in these units were passed on to the next generation.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:12]:
```suggestion
        The palace guards patrolled the capital's multiple palaces' gates, towers, and walls. They numbered around 2000 and were managed by the Minister of Guards, one of the nine ministers. The Minister of the Household commanded several guard squads as well. The two militarily most valuable were the “Rapid as Tigers” (which the Champion Spearman represents), and the “Feathered Forest”. Both were between 1000 and 2000 men strong and responsible for the emperor's immediate personal security. Memberships in these units were passed on to the next generation.
```

**Review by TheShadowOfHassen — COMMENTED** [review]
Copyedit

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 5]
```diff
@@ -6,6 +6,13 @@
     <SpecificName>Hǔ Bēn</SpecificName>
     <SelectionGroupName>units/han/champion_infantry_spearman</SelectionGroupName>
     <Icon>units/han/champion_infantry_swordsman.png</Icon>
+    <History>
+        The Han empire's capital was protected by various guard corps.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:10]:
```suggestion

```

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 6]
```diff
@@ -6,6 +6,13 @@
...
+        The largest and strategically most important was the Northern Army. It consisted of five regiments: the Archers, the Foot soldiers, the Elite Cavalry, the Garrison Cavalry, and the Chang River Regiment. Later, the army also contained non-Chinese auxiliary troops. Each regiment contained around 1,000 professionally trained soldiers. There was also a Southern Army of similar size, however, it didn't last as long and was of lower importance.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:11]:
```suggestion
       The Han empire's capital was protected by various guard corps. The largest and strategically most important was the Northern Army. It consisted of five regiments: the Archers, the Foot soldiers, the Elite Cavalry, the Garrison Cavalry, and the Chang River Regiment. Later, the army also contained non-Chinese auxiliary troops. Each regiment contained around 1,000 professionally trained soldiers. There was also a Southern Army of similar size, however, it didn't last as long and was of lower importance.
```

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 7]
```diff
@@ -6,6 +6,13 @@
...
+        The palace guards patrolled the capital's multiple palaces' gates, towers, and walls. They numbered around 2000 and were managed by the Minister of Guards, one of the nine ministers.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:12]:
```suggestion

```

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 8]
```diff
@@ -6,6 +6,13 @@
...
+        The Minister of the Household commanded several guard squads as well. The two militarily most valuable were the “Rapid as Tigers” (which the Champion Spearman represents), and the “Feathered Forest”. Both were between 1000 and 2000 men strong and responsible for the emperor's immediate personal security. Memberships in these units were passed on to the next generation.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:13]:
```suggestion
                The palace guards patrolled the capital's multiple palaces' gates, towers, and walls. They numbered around 2000 and were managed by the Minister of Guards, one of the nine ministers. The Minister of the Household commanded several guard squads as well. The two militarily most valuable were the “Rapid as Tigers” (which the Champion Spearman represents), and the “Feathered Forest”. Both were between 1000 and 2000 men strong and responsible for the emperor's immediate personal security. Memberships in these units were passed on to the next generation.

File: 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml [hunk 9]
```diff
@@ -6,6 +6,13 @@
...
+        Private areas were guarded by eunuchs, castrated men serving in imperial palaces. They were, on paper, under the command of the Minister Steward.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml:14]:
```suggestion
        Private areas were guarded by eunuchs, castrated men serving in imperial palaces. Who were, on paper, under the command of the Minister Steward.
```

**Review by TheShadowOfHassen — COMMENTED** [review]
Last changes I think for this one.

**Pull request merged**